### ISSUE TITLE
"no files to check" with sample `.pre-commit-config.yaml`

### ISSUE THREAD

**Issue by JMuff22** [jonasbb/pre-commit-latex-hooks#58]
Testing out this pre-commit repo with the sample config file given in the `README`. In my git top directory I have 1 `.tex` file however when I commit, pre-commit tells me that there are no files to check. Do I need to add something to the config file?

**Comment by jonasbb** [comment]
Hi, I cannot reproduce that. I created a new empty repository, added a `test.tex` and `.pre-commit-config.yaml`, and ran `pre-commit install --install-hooks`. With that, I get the expected messages when committing "broken" LaTeX. Maybe you could provide some more details, a list of filenames in your repository, the versions of pre-commit and its dependencies (`pip freeze`) and the operating system.

**Comment by JMuff22** [comment]
Hi, I think I have figured it out. It seems to be an error using VS Code conflicts with git? Unsure why. 

I did what you said, created an empty git repo etc and found no issues. I then replicated this in the repo that I was originally using and I had no issues as well. The next step I took then was to test the test repo in VS code and it reproduces the error. 

I updated my git extension through VS code and this seems to have resolved the issue. 

Thanks for responding and apologies for using your time on my mistake.
